pyseir/ensembles/ensemble_runner.py

```diff
@@ -24,6 +24,14 @@
 
 _logger = logging.getLogger(__name__)
 
+def _get_or_make_directory(*path): 
+    current_path_pieces = []
+    for path_piece in path: 
+        current_path = os.path.join(*current_path_pieces, path_piece)
+        if not os.path.exists(current_path): 
+            os.mkdir(current_path)
```

Comment:
I think you can just call `os.mkdirs(full_path, exists=True)` instead of needing to build this logic yourself?

Reply:
ah, it's `exist_ok`: https://docs.python.org/3/library/os.html#os.makedirs